src/components/Auth/ErrorNotificattion.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  errorMessage: string,
+  setErrorMessage: (param: string) => void;
+
+};
+
+export const ErroNotification: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage,
+}) => {
+  const [hiddenError, setHiddenError] = useState(false);
+
+  useEffect(() => {
+    const showMessage = setTimeout(() => {
+      setErrorMessage('');
+      if (hiddenError) {
+        clearInterval(showMessage);
+      }
+    }, 3000);
+  }, [errorMessage]);
```

Comment:
👍 

Also it could be this way(with ComponentWillUnmount realization) 👇 
```suggestion
  useEffect(() => {
    const timer = setTimeout(() => setErrorMessage('') , 3000);
    
     return () => {
      clearTimeout(timer);
    };
  }, [errorMessage]);
```

Reply:
блок з if мене трохи збив)